scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -0,0 +1,733 @@
+
+    #Create runtimeClass object
+    oc apply -f - <<EOF
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: kata-$tee_type
+handler: kata-$tee_type
+overhead:
+  podFixed:
+    memory: "350Mi"
+    cpu: "250m"
+    $ext_resources
+scheduling:
+  nodeSelector:
+     $label
+EOF
+    result=$?
+    [ $result -eq 0 ] || return 1
+
+    echo "kata-$tee_type RuntimeClass object successfully created"
+    return 0
+
+}
+
+function display_help() {
+    echo "Usage: install.sh -t <tee_type> [-h] [-m] [-s] [-b] [-u]"
+    echo "Options:"
+    echo "  -t <tee_type> Specify the TEE type (tdx or snp)"
+    echo "  -h Display help"
+    echo "  -m Install the image mirroring config"
+    echo "  -s Set additional cluster-wide image pull secret."
+    echo "     Requires the secret to be set in PULL_SECRET_JSON environment variable"
+    echo "     Example PULL_SECRET_JSON='{\"my.registry.io\": {\"auth\": \"ABC\"}}'"
+    echo "  -b Use pre-ga operator bundles"
+    echo "  -u Uninstall the installed artifacts"
+    echo " "
+    echo "Some environment variables that can be set:"
+    echo "SKIP_NFD: Skip NFD operator installationa and CR creation (default: false)"
+    echo "TRUSTEE_URL: Trustee URL to be used in the kernel config (default: http://kbs-service:8080)"
+    echo "CMD_TIMEOUT: Timeout for the commands (default: 900)"
+    # Add some example usage options
+    echo " "
+    echo "Example usage:"
+    echo "# Install the GA operator for snp"
+    echo " ./install.sh -t snp"
+    echo " "
+    echo "# Install the GA operator for tdx"
+    echo " ./install.sh -t tdx"
+    echo " "
+    echo "# Install the GA operator with image mirroring for snp"
+    echo " ./install.sh -m -t snp"
+    echo " "
+    echo "# Install the GA operator with additional cluster-wide image pull secret for tdx"
+    echo " export PULL_SECRET_JSON='{"brew.registry.redhat.io": {"auth": "abcd1234"}, "registry.redhat.io": {"auth": "abcd1234"}}'"
+    echo " ./install.sh -s -t tdx"
+    echo " "
+    echo "# Install the pre-GA operator with image mirroring and additional cluster-wide image pull secret for snp"
+    echo " ./install.sh -m -s -b -t snp"
+    echo " "
+    echo "# Deploy the pre-GA OSC operator with image mirroring and additional cluster-wide image pull secret for tdx"
+    echo " export PULL_SECRET_JSON='{"brew.registry.redhat.io": {"auth": "abcd1234"}, "registry.redhat.io": {"auth": "abcd1234"}}'"
+    echo " ./install.sh -m -s -b -t tdx"
+    echo " "
+    echo "# Uninstall the installed artifacts for tdx"
+    echo " ./install.sh -t tdx -u"
+    echo " "
+    echo "# Uninstall the installed artifacts for snp"
+    echo " ./install.sh -t snp -u"
+    echo " "
+}
+
+# Function to verify all required variables are set and
+# required files exist
+
+function verify_params() {
+
+    # Check if TEE_TYPE is provided
+    if [ -z "$TEE_TYPE" ]; then
+        echo "Error: TEE type (-t) is mandatory"
+        display_help
+        exit 1
```

Comment:
What about `return 1` and let the caller decide ?

Reply:
Yeah, I'll make it consistent across the code.